src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -15,59 +15,154 @@ namespace DSCoreNodesUI
     [IsDesignScriptCompatible]
     public class DynamoConvert : NodeModel
     {
-        private ConversionUnit selectedFromConversion;
-        private ConversionUnit selectedToConversion;
-       
-        public ConversionUnit SelectedFromConversion
+        private object selectedFromConversion;
+        private object selectedToConversion;
+        private object selectedMetricConversion;
+        private bool isSelectionFromBoxEnabled;
+        private string selectionFromBoxToolTip;
+
+        private object selectedFromConversionSource;
+
+        public object SelectedFromConversionSource
+        {
+            get { return selectedFromConversionSource; }
+            set
+            {
+                selectedFromConversionSource = value;
+                RaisePropertyChanged("SelectedFromConversionSource");
+            }
+        }
+
+        private object selectedToConversionSource;
+        public object SelectedToConversionSource
```

Comment:
@ramramps The DynamoConversions project needs to be moved to CoreNodeModelsWpf and updated for both VS2012 and VS2013. The references to things like System.xaml that were added to support this library in CoreNodeModels then need to be removed as well.